src/content/reference/okteto-cli.mdx

```diff
@@ -268,17 +269,27 @@ If you need to destroy external resources (like s3 buckets or other Cloud resour
 $ okteto destroy
 ```
 
+The `okteto destroy` command will only destroy resources matching the specified environment name. In a monorepo scenario, where you might have several manifests, you need to specify the correct manifest to destroy the specific development environment. This ensures that the appropriate destroy commands, as defined in your Okteto manifest, are executed within a particular scope.
```

Comment:
```suggestion
The `okteto destroy` command will only destroy resources matching the specified development environment name. In a monorepo scenario, where you might have several manifests, you need to specify the correct manifest to destroy the specific development environment. This ensures that the appropriate destroy commands, as defined in your Okteto manifest, are executed within a particular scope.
```
The more specific name is "development environment name".
I would also mention that, y default, this is the name of the folder. But you can specify the `name` in the okteto manifest, or with the `--name` flag